WorkflowReactiveSwift/TestingTests/TestingTests.swift

```diff
@@ -72,6 +72,15 @@ class WorkflowReactiveSwiftTestingTests: XCTestCase {
         }
     }
 
+    func test_worker_unexpected() {
+        let tester = TestWorkflow()
+            .renderTester(initialState: .init(mode: .worker(input: "test"), output: ""))
+
+        expectingFailure(#"Unexpected workflow of type WorkerWorkflow<TestWorker> with key """#) {
```

Comment:
Wait, what kind of wizardry is this?

Reply:
[SE-200](https://github.com/apple/swift-evolution/blob/master/proposals/0200-raw-string-escaping.md)!